Source/WebGPU/WebGPU/Instance.mm

```diff
@@ -196,6 +196,27 @@
     callback(WGPURequestAdapterStatus_Success, Adapter::create(sortedDevices[0], *this, options.xrCompatible, WTFMove(*deviceCapabilities)), { });
 }
 
+void Instance::retainDevice(Device& device, id<MTLCommandBuffer> commandBuffer)
+{
+    Locker locker(m_lock);
+    CommandBufferContainer* container = nullptr;
```

Comment:
I suppose these 5 lines could also read:
```
CommandBufferContainer& container = retainedDeviceInstances.add(&device, CommandBufferContainer { }).iterator->value;
```
but this always creates a new `CommandBufferContainer` and discards the result when it already exists. Not sure if this matters.